pkg/parser/expression_test.go

```diff
@@ -300,3 +304,76 @@ func TestParseTopLevelExpressionErr(t *testing.T) {
 		assert.Equal(t, wantErr, got.Error(), "input: %s\nerrors:\n%s", input, parser.errors)
 	}
 }
+
+func TestParseNiladic(t *testing.T) {
+	inputs := map[string]string{
+		"rand1":       "print rand1",
+		"rand1-twice": "print rand1 rand1",
+		"rand1-expr":  "print rand1+10",
+		"rand1-toplevelexpr": `
+n := rand1+0.5
+print n
+`,
+		"rand1-toplevelexpr-space": `
+if rand1 > 0.5
+  print "big"
```

Comment:
all the other test programs use a tab for indenting the block where there is one. this uses two spaces.

Reply:
great, done, thanks.